tests/v2/validation/provisioning/permutations/permutations.go

```diff
@@ -57,6 +58,14 @@ func RunTestPermutations(s *suite.Suite, testNamePrefix string, client *rancher.
 				s.Run(name, func() {
 					switch clusterType {
 					case RKE2ProvisionCluster, K3SProvisionCluster:
+						if testClusterConfig.CloudProvider == "aws" {
+							byteYaml, err := os.ReadFile("../resources/aws-out-of-tree.yaml")
```

Comment:
We're not able to see this yaml file, is it supposed to be something private or configurable?

Reply:
yaml is in the gitignore, so I'll rename to .yml so that you can see it.